mlir/lib/Conversion/VectorToXeGPU/VectorToXeGPU.cpp

```diff
@@ -165,6 +496,20 @@ struct TransferReadLowering : public OpRewritePattern<vector::TransferReadOp> {
     if (failed(transferPreconditions(rewriter, readOp)))
       return failure();
 
+    auto chip = xegpu::getXeGPUChipStr(readOp);
+    if ( chip != "pvc" && chip != "bmg") {
```

Comment:
So the logical is transferReadOp will be lowered into LoadGather for PVC amd bmg. and to LoadNd for others?

Reply:
If the target is "PVC" or "BMG", lower the transfer_read to 2d block load/store. For other targets, lower it to scattered load/store.